packages/kotti-ui/source/kotti-value-label/KtValueLabel.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+	<div :class="rootClasses" :data-test="dataTest">
+		<template v-if="isLoading">
+			<div
+				v-if="hasLabel || hasHelpText || $slots.helpText"
```

Comment:
since it's repeated in line 5 and 8, i'd refact into a `showLabel` computed value